detect_secrets/plugins/base.py

```diff
@@ -53,19 +53,33 @@ class BasePlugin(object):
     def secret_type(self):
         raise NotImplementedError
 
-    def __init__(self, exclude_lines_regex=None, should_verify=False, **kwargs):
+    def __init__(
+        self,
+        exclude_lines_regex=None,
+        should_verify=False,
+        false_positive_heuristics=None,
+        **kwargs
+    ):
         """
         :type exclude_lines_regex: str|None
         :param exclude_lines_regex: optional regex for ignored lines.
 
         :type should_verify: bool
+
+        :type false_positive_heuristics: List[Callable]|None
+        :param false_positive_heuristics: List of fp-heuristic functions
+        applicable to this plugin
         """
         self.exclude_lines_regex = None
         if exclude_lines_regex:
             self.exclude_lines_regex = re.compile(exclude_lines_regex)
 
         self.should_verify = should_verify
 
+        self.false_positive_heuristics = false_positive_heuristics \
+            if false_positive_heuristics \
+            else []
```

Comment:
If the only use is `is_secret_false_positive` and that has an `if self.false_positive_heuristics` predicate, we probably don't need the `\`
`\`
since it'll be None by default.

If there is a different reason, maybe we can do
```python
self.false_positive_heuristics = (
    false_positive_heuristics
    if false_positive_heuristics
    else []
)
```